scripts/cluster_network.py

```diff
@@ -349,7 +352,7 @@ def plot_busmap_for_n_clusters(n, n_clusters, fn=None):
 
     if snakemake.wildcards.clusters.endswith('m'):
         n_clusters = int(snakemake.wildcards.clusters[:-1])
-        aggregate_carriers = pd.Index(n.generators.carrier.unique()).difference(renewable_carriers)
+        aggregate_carriers = None
```

Comment:
This will break the intention of this method. Either keep as before, or change to:
```suggestion
        aggregate_carriers = snakemake.config["electricity"].get("conventional_carriers")
```
However, with my suggestion `ror` would be also higher resolved. The old version did cluster `ror` to the same resolution as conventional plants. But this might have been a minor bug, now that I think of it.

Reply:
Also, I see that there's a bug in case `aggregate_carriers` is an empty list. But theoretically, you *could* run a simulation without conventional carriers and want to use the `m`-wildcard option.... I'll put that in an issue. :drop:

Reply:
great, thanks for spotting that